test/parallel/test-domain-add-remove.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
+
+const common = require('../common');
```

Comment:
I think you can drop the left hand side of this assignment, since `common` isn't actually used anywhere.

Reply:
Thanks. I forgot to run the linter before opening the pull request. Fixed this and switched from `notEqual` to `notStrictEqual` further down this file.